gulpfile.js

```diff
@@ -50,7 +58,7 @@ gulp.task('make-config', function () {
 });
 
 gulp.task('flavour', function () {
-	var flavour = process.env.FLAVOUR || 'vimojo';
+	let flavour = process.env.FLAVOUR || 'vimojo';
```

Comment:
do we have ES6 transpile support into ES5? If not, we should keep var instead of let